ADOPTERS.md

```diff
@@ -0,0 +1,5 @@
+| Organization                                                             | Contact                                                                                                                                                                                                      | Description of Use                                                                                                                                                                                                                                                                                                                                   |
+| ------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Gitpod](https://www.gitpod.io)                                          | [@gitpod](https://www.twitter.com/gitpod)                                                                                                                                                                    | Gitpod builds Gitpod with Gitpod on Gitpod.                                                                                                                                                                                                                                                                                                          |
+| <the name of your company and a link to the website>                     | <a name or handle of a person and link to a social media account>                                                                                                                                            | <describe how you are using Gitpod>                                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| [GitLab](https://gitlab.com)                                             | [@gitlab](https://www.twitter.com/gitlab)                                                                                                                                                                    | Developing and reviewing changes for GitLab, the handbook, marketing page and documentation.                                                                                                                                                                                                                                                        |
| <the name of your company and a link to the website>                     | <a name or handle of a person and link to a social media account>                                                                                                                                            | <describe how you are using Gitpod>                                                                                                                                                                                                                                                                                                                  |
``` 

@ghuntley Love the idea, how about already adding GitLab as first adopter here? 😉